README.md

```diff
@@ -113,6 +113,66 @@ next-page-title: "Create a Layout"
 Omit the "prev-page" info for the first page, and the "next-page" info for the
 last page.
 
+## Syntax highlighting
+
+The flutter.io website uses [prism.js](http://prismjs.com/) for syntax
+highlighting. This section covers how to use syntax highlighting, and
+how to update our syntax highlighter for new languages.
+
+### Using syntax highlighting
+
+The easiest way to syntax highlight a block of code is to wrap
+it with `{% prettify _language_ %}`:
+
+```
+{% prettify dart %}
+class SomeCode {
+  String name;
+}
+{% endprettify %}
+```
+
+The _prettify_ tag converts the block into `<code>` and `<pre>`, and
+applies the necessary CSS classes for Prism to turn on syntax highlighting.
+You must specify a language for syntax highlighting to work properly.
+
+We don't currently have a good way to syntax highlight inline code. It may be
+possible, if you know Prism's CSS classes. "It's an exercise left up
+to the reader."
+
+### Supported languages
+
+This website can syntax highlight the following languages:
```

Comment:
Can we list the actual values you need to enter in the markdown for `_language_` for each of these?